cli/module_generate/templates/python/tmpl-requirements.txt

```diff
@@ -1,2 +1,5 @@
 viam-sdk=={{ .SDKVersion }}
 typing-extensions
+{{ if eq .ResourceSubtype "mlmodel" }}
+viam-sdk[mlmodel]
+{{ end }}
```

Comment:
I think this is an if/else: either we do `viam-sdk` or we do `viam-sdk[mlmodel]`, but i don't think we need both?